packages/stop-viewer-overlay/package.json

```diff
@@ -20,13 +20,13 @@
   },
   "dependencies": {
     "@opentripplanner/base-map": "^3.1.0",
-    "@opentripplanner/core-utils": "^11.4.0"
+    "@opentripplanner/core-utils": "^11.4.1"
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.